templates/takeovers/_ai_webinar-2018-10-01.html

```diff
@@ -17,7 +17,7 @@ <h1 class="p-takeover__title">How to build and deploy your first AI/ML model on
         <img class="server-picto" src="{{ ASSET_SERVER_URL }}fd20f946-server-icon.svg" alt="">
       </div>
       <div class="u-hide--medium u-hide--large">
-        <a href="https://www.brighttalk.com/webcast/6793/336267" class="p-button--positive u-no-margin--bottom">Watch on-demand</a>
+        <a href="https://www.brighttalk.com/webcast/6793/336267" class="p-button--positive u-no-margin--bottom">Register for the webinar</a>
```

Comment:
Please use the landing page for the URL: `/engage/deploy-ai-ml-model?utm_medium=takeover&utm_campaign=FY19_Cloud_K8_WBN_AIWebinar2`